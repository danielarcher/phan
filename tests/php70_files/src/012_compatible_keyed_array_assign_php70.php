<?php 
$data = [
    [1, 'Foo'],
    [2, 'Bar'],
];

[$id1, $name1] = $data[0];

foreach ($data as [$id, $name]) {
    
}
