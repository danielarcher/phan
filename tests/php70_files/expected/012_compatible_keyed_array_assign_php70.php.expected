%s:8 PhanCompatibleShortArrayAssignPHP70 Square bracket syntax for an array destructuring assignment is not compatible with PHP 7.0
%s:11 PhanCompatibleShortArrayAssignPHP70 Square bracket syntax for an array destructuring assignment is not compatible with PHP 7.0
